internal/image.go

```diff
@@ -29,13 +30,19 @@ func NewImage(pdf gofpdf.Pdf, math Math) *image {
 }
 
 // AddFromFile open an image from disk and add to PDF
-func (s *image) AddFromFile(path string, marginTop float64, indexCol float64, qtdCols float64, colHeight float64, prop props.Rect) {
+func (s *image) AddFromFile(path string, marginTop float64, indexCol float64, qtdCols float64, colHeight float64, prop props.Rect) (err error) {
```

Comment:
Go style note: I would suggest not to use this idiom (named return values in function signature + naked returns in function body). They are confusing and error-prone. I would suggest to always explicitly return what needs to be returned, no matter how small the function is.

Reply:
Make sense, I will change it :D